src/jobs/generation/MetaGenerator.groovy

```diff
@@ -394,7 +394,7 @@ repos.each { repoInfo ->
                     // If not a PR, then the branch name should be the target branch
                     if (isPRTest) {
                         if (isVSTS) {
-                            // TODO: VSTS PR branch
+                            branch("*/${repoInfo.branch}")
```

Comment:
This needs to be the hash that is being PR'd.

Reply:
Changed it to vstsBranch, which is set by TFS Jenkins plugin with the commit hash that is being PR'd.